### ISSUE TITLE
Wrong timing of Shotgun/Rifle "reload2" sequence ammo counting

### ISSUE THREAD

**Issue by DoofusCockslap** [zm-reborn/zmr-game#79]
In my opinion reloading sequence for shotgun and rifle in ZM:R feels a bit weird and confusing. Maybe because in comparison to 1.2.1, in ZM:R ammo information is being updated only after the "reload2" animation is finished playing, which feels very confusing and may bug out sometimes. In 1.2.1 on other hand, it works as it should in any other Source game where ammo information is updated when the "reload2" animation starts playing.
The old variant is simply more comfortable than getting used to the new way of ammo counting.

**Comment by TotallyMehis** [comment]
what

**Comment by DoofusCockslap** [comment]
This one would be difficult to explain, but let me try again. When you're reloading a shotgun in HL2 it normally updates the ammo count information instantly when "reload2" animation sequence starts playing. In ZMR it's a bit different, since the ammo count information is being updated only when "reload2" animation sequence finished playing. It feels kinda delayed compared to how it normally works in other Source games and mods.
